lib/backend/k8s/conversion_test.go

```diff
@@ -702,41 +702,31 @@ var _ = Describe("Test Node conversion", func() {
 				Name:            "TestNode",
 				Labels:          l,
 				ResourceVersion: "1234",
-			},
-			Spec: k8sapi.NodeSpec{},
-			Status: k8sapi.NodeStatus{
-				Addresses: []k8sapi.NodeAddress{
```

Comment:
Might want to leave some of these other IPs in, just for funsies, and to make sure we don't do anything weird if they're present (it's more realistic)